### ISSUE TITLE
Legend accessibility "The SVG element does not have accessible name"

### ISSUE THREAD

**Issue by tekno0ryder** [recharts/recharts#5412]
## Do you want to request a feature or report a bug?

Accessibility enhancement.

## What is the current behavior?

https://recharts.org/en-US/api/Legend

<img width="1431" alt="Screenshot 2025-01-02 at 2 16 29 PM" src="https://github.com/user-attachments/assets/2744bf70-3fd4-4fe8-a10a-e329e676ce1f" />

## What is the expected behavior?

Legend svg icon should have role none or presentation.

## Which versions of Recharts, and which browser / OS are affected by this issue? Did this work in previous versions of Recharts?

Tested on chrome and version 2.13.0 and also on the deployed version in recharts website

**Comment by ckifer** [comment]
https://github.com/recharts/recharts/pull/5414

**Comment by tekno0ryder** [comment]
Thanks for addressing the issue, very fast!

**Comment by ckifer** [comment]
Going to close this, it'll be in 3.0 and is out in the alpha now
